golang.req

```diff
@@ -21,6 +33,9 @@ end
 requires = []
 
 unless filelist.empty? then
+
+	requires << "golang(API) = " + go_get_version()
```

Comment:
That is still wrong, We don't have any runtime requirement for that, it's always buildtime requirement only.